### ISSUE TITLE
Conda 4.7.5 corrupt tarball can't install packages

### ISSUE THREAD

**Issue by Ruler26** [conda/conda#8935]
## Current Behavior
After conda update --all a couple of days ago I lost a lot of packages (probably due to issue #8842  ). I've been steadily re-installing my packages. However I've hit the issue that the conda tarball seems to be corrupt:

I've tried manually removing the file, but to no avail.

```
(base) D:\>conda install dask
Collecting package metadata (current_repodata.json): done
Solving environment: done
WARNING conda.core.package_cache_data:_make_single_record(350): Encountered corr
upt package tarball at D:\Anaconda\pkgs\conda-4.7.5-py37_0.conda. Conda has remo
ved it, but you need to re-run conda to download it again.

## Package Plan ##

  environment location: D:\Anaconda

  added / updated specs:
    - dask


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    conda-4.7.5                |           py37_0         2.8 MB
    ------------------------------------------------------------
                                           Total:         2.8 MB

The following NEW packages will be INSTALLED:

  bokeh              pkgs/main/win-64::bokeh-1.2.0-py37_0
  click              pkgs/main/win-64::click-7.0-py37_0
  cytoolz            pkgs/main/win-64::cytoolz-0.9.0.1-py37hfa6e2cd_1
  dask               pkgs/main/noarch::dask-2.1.0-py_0
  dask-core          pkgs/main/noarch::dask-core-2.1.0-py_0
  distributed        pkgs/main/noarch::distributed-2.1.0-py_0
  heapdict           pkgs/main/win-64::heapdict-1.0.0-py37_2
  libtiff            pkgs/main/win-64::libtiff-4.0.10-hb898794_2
  locket             pkgs/main/win-64::locket-0.2.0-py37_1
  msgpack-python     pkgs/main/win-64::msgpack-python-0.6.1-py37h74a9793_1
  olefile            pkgs/main/win-64::olefile-0.46-py37_0
  partd              pkgs/main/noarch::partd-1.0.0-py_0
  pillow             pkgs/main/win-64::pillow-6.0.0-py37hdc69c19_0
  pyyaml             pkgs/main/win-64::pyyaml-5.1.1-py37he774522_0
  sortedcontainers   pkgs/main/win-64::sortedcontainers-2.1.0-py37_0
  tblib              pkgs/main/noarch::tblib-1.4.0-py_0
  tk                 pkgs/main/win-64::tk-8.6.8-hfa6e2cd_0
  toolz              pkgs/main/win-64::toolz-0.9.0-py37_0
  zict               pkgs/main/noarch::zict-1.0.0-py_0

The following packages will be SUPERSEDED by a higher-priority channel:

  ca-certificates    conda-forge::ca-certificates-2019.6.1~ --> pkgs/main::ca-ce
rtificates-2019.5.15-0
  certifi                                       conda-forge --> pkgs/main
  conda                                         conda-forge --> pkgs/main


Proceed ([y]/n)? y


Downloading and Extracting Packages
conda-4.7.5          | 2.8 MB    | #################################### | 100%

InvalidArchiveError('Error with archive D:\\Anaconda\\pkgs\\conda-4.7.5-py37_0.c
onda.  You probably need to delete and re-download or re-create this file.  Mess
age from libarchive was:\n\nCould not unlink (errno=22, retcode=-25, archive_p=4
06752012528)')
```

### Steps to Reproduce
`conda install dask`
OR
`conda update conda`
OR
many other packages with a dependency with conda 4.7.5

## Expected Behavior
I need to be able to install conda 4.7.5 and other packages

## Environment Information
<details open><summary><code>`conda info`</code></summary><p>

```
     active environment : base
    active env location : D:\Anaconda
            shell level : 1
       user config file : C:\Users\r.est.ds.admin\.condarc
 populated config files : C:\Users\r.est.ds.admin\.condarc
          conda version : 4.7.5
    conda-build version : not installed
         python version : 3.7.3.final.0
       virtual packages :
       base environment : D:\Anaconda  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
                          https://conda.anaconda.org/conda-forge/win-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : D:\Anaconda\pkgs
                          C:\Users\r.est.ds.admin\.conda\pkgs
                          C:\Users\r.est.ds.admin\AppData\Local\conda\conda\pkgs

       envs directories : D:\Anaconda\envs
                          C:\Users\r.est.ds.admin\.conda\envs
                          C:\Users\r.est.ds.admin\AppData\Local\conda\conda\envs

               platform : win-64
             user-agent : conda/4.7.5 requests/2.22.0 CPython/3.7.3 Windows/2012
ServerR2 Windows/6.3.9600
          administrator : True
             netrc file : None
           offline mode : False
```
</p></details>

<details open><summary><code>`conda config --show-sources`</code></summary><p>

```
(base) D:\>conda config --show-sources
==> C:\Users\r.est.ds.admin\.condarc <==
ssl_verify: True
channels:
  - defaults
  - conda-forge
```
</p></details>

<details><summary><code>`conda list --show-channel-urls`</code></summary><p>

```
(base) D:\>conda list --show-channel-urls
# packages in environment at D:\Anaconda:
#
# Name                    Version                   Build  Channel
alabaster                 0.7.12                   py37_0    defaults
anaconda                  0.0.1.1                  pypi_0    pypi
asn1crypto                0.24.0                   py37_0    defaults
astroid                   2.2.5                    py37_0    defaults
attrs                     19.1.0                   py37_1    defaults
babel                     2.7.0                      py_0    defaults
backcall                  0.1.0                    py37_0    defaults
blas                      1.0                         mkl    defaults
bleach                    3.1.0                    py37_0    defaults
bokeh                     0.13.0                   pypi_0    pypi
boto                      2.49.0                   py37_0    defaults
boto3                     1.7.31                   pypi_0    pypi
botocore                  1.10.31                  pypi_0    pypi
bz2file                   0.98                     pypi_0    pypi
bzip2                     1.0.7                he774522_0    defaults
ca-certificates           2019.6.16            hecc5488_0    conda-forge
certifi                   2019.6.16                py37_0    conda-forge
cffi                      1.12.3           py37h7a1dbc1_0    defaults
chardet                   3.0.4                    py37_1    defaults
cloudpickle               1.2.1                      py_0    defaults
colorama                  0.4.1                    py37_0    defaults
colorlover                0.2.1                    pypi_0    pypi
conda                     4.7.5                    py37_0    conda-forge
conda-package-handling    1.3.10                   py37_0    defaults
cprint                    1.1                      pypi_0    pypi
cryptography              2.7              py37h7a1dbc1_0    defaults
cufflinks                 0.14.4                   pypi_0    pypi
cycler                    0.10.0                   py37_0    defaults
cymem                     2.0.2            py37h74a9793_0    defaults
cython-blis               0.2.4            py37hfa6e2cd_0    conda-forge
decorator                 4.4.0                    py37_1    defaults
deduce                    1.0.1                    pypi_0    pypi
defusedxml                0.6.0                      py_0    defaults
dill                      0.2.8.2                  pypi_0    pypi
docutils                  0.14                     py37_0    defaults
docx2txt                  0.7                      pypi_0    pypi
eli5                      0.8.2                    pypi_0    pypi
entrypoints               0.3                      py37_0    defaults
et_xmlfile                1.0.1                    py37_0    defaults
freetype                  2.9.1                ha9979f8_1    defaults
funcy                     1.12                     pypi_0    pypi
gensim                    3.4.0                    pypi_0    pypi
geographiclib             1.49                     pypi_0    pypi
geopy                     1.14.0                   pypi_0    pypi
graphviz                  2.38                 hfd603c8_2    defaults
httplib2                  0.11.3                   pypi_0    pypi
icc_rt                    2019.0.0             h0cc432a_1    defaults
icu                       58.2                 ha66f8fd_1    defaults
idna                      2.8                      py37_0    defaults
imagesize                 1.1.0                    py37_0    defaults
imbalanced-learn          0.4.3                    pypi_0    pypi
imblearn                  0.0                      pypi_0    pypi
intel-openmp              2019.4                      245    defaults
ipykernel                 5.1.1            py37h39e3cac_0    defaults
ipython                   7.6.0            py37h39e3cac_0    defaults
ipython_genutils          0.2.0                    py37_0    defaults
ipywidgets                7.4.2                    py37_0    defaults
isodate                   0.6.0                    pypi_0    pypi
isort                     4.3.21                   py37_0    defaults
jdcal                     1.4.1                      py_0    defaults
jedi                      0.13.3                   py37_0    defaults
jinja2                    2.10.1                   py37_0    defaults
jmespath                  0.9.3                    pypi_0    pypi
joblib                    0.12.0                   pypi_0    pypi
jpeg                      9b                   hb83a4c4_2    defaults
jsonschema                3.0.0a3               py37_1000    conda-forge
jupyter                   1.0.0                    py37_7    defaults
jupyter_client            5.2.4                    py37_0    defaults
jupyter_console           6.0.0                    py37_0    defaults
jupyter_core              4.5.0                      py_0    defaults
keyring                   18.0.0                   py37_0    defaults
kiwisolver                1.1.0            py37ha925a31_0    defaults
lazy-object-proxy         1.4.1            py37he774522_0    defaults
libarchive                3.3.3                h0643e63_5    defaults
libiconv                  1.15                 h1df5818_7    defaults
libpng                    1.6.37               h2a8f88b_0    defaults
libsodium                 1.0.16               h9d3ae62_0    defaults
libxml2                   2.9.9                h464c3ec_0    defaults
lxml                      4.3.4                    pypi_0    pypi
lz4-c                     1.8.1.2              h2fa13f4_0    defaults
lzo                       2.10                 h6df0209_2    defaults
m2w64-gcc-libgfortran     5.3.0                         6    defaults
m2w64-gcc-libs            5.3.0                         7    defaults
m2w64-gcc-libs-core       5.3.0                         7    defaults
m2w64-gmp                 6.1.0                         2    defaults
m2w64-libwinpthread-git   5.0.0.4634.697f757               2    defaults
markupsafe                1.1.1            py37he774522_0    defaults
matplotlib                3.0.1                    pypi_0    pypi
matplotlib-venn           0.11.5                   pypi_0    pypi
mccabe                    0.6.1                    py37_1    defaults
menuinst                  1.4.16           py37he774522_0    defaults
mistune                   0.8.4            py37he774522_0    defaults
mkl                       2019.4                      245    defaults
mkl-service               2.0.2            py37he774522_0    defaults
mkl_fft                   1.0.12           py37h14836fe_0    defaults
mkl_random                1.0.2            py37h343c172_0    defaults
mpi4py                    3.0.0                    pypi_0    pypi
msgpack                   0.6.1                    pypi_0    pypi
msys2-conda-epoch         20160418                      1    defaults
murmurhash                0.28.0                   pypi_0    pypi
nbconvert                 5.5.0                      py_0    defaults
nbformat                  4.4.0                    py37_0    defaults
nl-core-news-sm           2.1.0                    pypi_0    pypi
nltk                      3.4.1                    py37_0    defaults
notebook                  5.7.8                    py37_0    defaults
numpy                     1.16.4           py37h19fb1c0_0    defaults
numpy-base                1.16.4           py37hc3f5095_0    defaults
numpydoc                  0.9.1                      py_0    defaults
openpyxl                  2.6.2                      py_0    defaults
openssl                   1.1.1c               he774522_1    defaults
packaging                 19.0                     py37_0    defaults
pandas                    0.24.2           py37ha925a31_0    defaults
pandoc                    2.2.3.2                       0    defaults
pandocfilters             1.4.2                    py37_1    defaults
parso                     0.5.0                      py_0    defaults
patsy                     0.5.1                    py37_0    defaults
pickleshare               0.7.5                    py37_0    defaults
pillow                    6.1.0                    pypi_0    pypi
pip                       19.1.1                   py37_0    defaults
plac                      0.9.6                    py37_0    defaults
plotly                    3.3.0                    pypi_0    pypi
preshed                   2.0.1            py37h33f27b4_0    defaults
progressbar               2.5                      pypi_0    pypi
prometheus_client         0.7.1                      py_0    defaults
prompt_toolkit            2.0.9                    py37_0    defaults
psutil                    5.6.3            py37he774522_0    defaults
pycodestyle               2.5.0                    py37_0    defaults
pycosat                   0.6.3            py37hfa6e2cd_0    defaults
pycparser                 2.19                     py37_0    defaults
pydot                     1.4.1                    pypi_0    pypi
pydotplus                 2.0.2                    py37_1    defaults
pyflakes                  2.1.1                    py37_0    defaults
pygments                  2.4.2                      py_0    defaults
pyldavis                  2.1.2                    pypi_0    pypi
pylint                    2.3.1                    py37_0    defaults
pynlpl                    1.2.7                    pypi_0    pypi
pyodbc                    4.0.26           py37ha925a31_0    defaults
pyopenssl                 19.0.0                   py37_0    defaults
pyparsing                 2.4.0                      py_0    defaults
pyqt                      5.9.2            py37h6538335_2    defaults
pyreadline                2.1                      pypi_0    pypi
pyrsistent                0.14.11          py37he774522_0    defaults
pyrtf3                    0.47.5                   pypi_0    pypi
pysocks                   1.7.0                    py37_0    defaults
python                    3.7.3                h8c8aaf0_1    defaults
python-dateutil           2.8.0                    py37_0    defaults
python-docx               0.8.10                   pypi_0    pypi
python-graphviz           0.8.3                    pypi_0    pypi
python-libarchive-c       2.8                     py37_10    defaults
pytz                      2019.1                     py_0    defaults
pywin32                   223              py37hfa6e2cd_1    defaults
pywinpty                  0.5.5                 py37_1000    defaults
pyzmq                     18.0.0           py37ha925a31_0    defaults
qt                        5.9.7            vc14h73c81de_0    defaults
qtawesome                 0.5.7                    py37_1    defaults
qtconsole                 4.5.1                      py_0    defaults
qtpy                      1.8.0                      py_0    defaults
rdflib                    4.2.2                    pypi_0    pypi
requests                  2.22.0                   py37_0    defaults
retrying                  1.3.3                    py37_2    defaults
rope                      0.14.0                     py_0    defaults
ruamel_yaml               0.15.46          py37hfa6e2cd_0    defaults
s3transfer                0.1.13                   pypi_0    pypi
scikit-learn              0.21.2           py37h6288b17_0    defaults
scikit-optimize           0.5.2                    pypi_0    pypi
scipy                     1.2.1            py37h29ff71c_0    defaults
seaborn                   0.9.0                    py37_0    defaults
send2trash                1.5.0                    py37_0    defaults
setuptools                41.0.1                   py37_0    defaults
sip                       4.19.8           py37h6538335_0    defaults
six                       1.12.0                   py37_0    defaults
smart-open                1.5.7                    pypi_0    pypi
smart_open                1.8.4                      py_0    defaults
snowballstemmer           1.9.0                      py_0    defaults
spacy                     2.1.4            py37he980bc4_0    conda-forge
spacy-model-en_core_web_sm 2.1.0              pyh57f5928_0    conda-forge
sphinx                    2.1.2                      py_0    defaults
sphinxcontrib-applehelp   1.0.1                      py_0    defaults
sphinxcontrib-devhelp     1.0.1                      py_0    defaults
sphinxcontrib-htmlhelp    1.0.2                      py_0    defaults
sphinxcontrib-jsmath      1.0.1                      py_0    defaults
sphinxcontrib-qthelp      1.0.2                      py_0    defaults
sphinxcontrib-serializinghtml 1.1.3                      py_0    defaults
spyder                    3.3.5                    py37_0    defaults
spyder-kernels            0.5.0                    py37_0    defaults
sqlalchemy                1.3.5            py37he774522_0    defaults
sqlite                    3.28.0               he774522_0    defaults
srsly                     0.0.7            py37h6538335_0    conda-forge
statsmodels               0.10.0           py37h8c2d366_0    defaults
tabulate                  0.8.3                    pypi_0    pypi
termcolor                 1.1.0                    py37_1    defaults
terminado                 0.8.2                    py37_0    defaults
testpath                  0.4.2                    py37_0    defaults
thinc                     7.0.4            py37he980bc4_0    conda-forge
tornado                   6.0.3            py37he774522_0    defaults
tqdm                      4.23.4                   pypi_0    pypi
traitlets                 4.3.2                    py37_0    defaults
treetaggerwrapper         2.2.4                    pypi_0    pypi
unidecode                 1.0.22                   pypi_0    pypi
urllib3                   1.24.2                   py37_0    defaults
vc                        14.1                 h0510ff6_4    defaults
vs2015_runtime            14.15.26706          h3a45250_4    defaults
wasabi                    0.2.2                      py_0    conda-forge
wcwidth                   0.1.7                    py37_0    defaults
webencodings              0.5.1                    py37_1    defaults
wheel                     0.33.4                   py37_0    defaults
widgetsnbextension        3.4.2                    py37_0    defaults
win_inet_pton             1.1.0                    py37_0    defaults
wincertstore              0.2                      py37_0    defaults
winpty                    0.4.3                         4    defaults
wrapt                     1.11.2           py37he774522_0    defaults
xlrd                      1.2.0                    py37_0    defaults
xz                        5.2.4                h2fa13f4_4    defaults
yaml                      0.1.7                hc54c509_2    defaults
zeromq                    4.3.1                h33f27b4_3    defaults
zlib                      1.2.11               h62dcd97_3    defaults
zstd                      1.3.7                h508b16e_0    defaults
```
</p></details>

**Comment by soapy1** [comment]
Seems like this should be fixed by clearing out your package cache. Try `conda clean --all`
